projects/components/src/datagrid/interfaces/datagrid-column.interface.ts

```diff
@@ -114,6 +114,15 @@ export enum ContextualButtonPosition {
     ROW = 'ROW',
 }
 
+/**
+ * An enum that describes the possible ways to display the button title.
+ */
+export enum ButtonTitleDisplay {
+    ICON = 'ICON',
+    LABEL = 'LABEL',
+    ICON_AND_LABEL = 'ICON_AND_LABEL',
```

Comment:
So I redefined these enums in terms of Bitmasks. ICON_AND_LABEL still needs to stick around because when we are set to ICON_AND_LABEL, we specifically don't want to show the tooltip, which is an important case.